zenoh_ros_type/autoware_auto_msgs/__init__.py

```diff
@@ -1,2 +0,0 @@
-from .autoware_auto_control_msgs import *
```

Comment:
I think you can still reserve the old message type, but just use the new message type in the latest FMS.

Reply:
These messages have been restored in the latest commit.